syntax/odin.vim

```diff
@@ -4,21 +4,23 @@ vim9script
 # Language: Odin
 # Maintainer: Maxim Kim <habamax@gmail.com>
 # Website: https://github.com/habamax/vim-odin
-# Last Change: 2024-01-15
+# Last Change: 2025-03-28
 
 if exists("b:current_syntax")
   finish
 endif
 
-syntax keyword odinKeyword using transmute cast distinct opaque where dynamic
+syntax keyword odinKeyword using transmute cast auto_cast distinct opaque where dynamic
 syntax keyword odinKeyword struct enum union const bit_field bit_set
 syntax keyword odinKeyword package proc map import export foreign
 syntax keyword odinKeyword size_of offset_of type_info_of typeid_of type_of align_of
 syntax keyword odinKeyword return defer
 syntax keyword odinKeyword or_return or_else
 syntax keyword odinKeyword inline no_inline
+syntax keyword odinKeyword asm context
```

Comment:
I can't find `asm` and `context` in odin docs, is there a link?

Reply:
https://github.com/odin-lang/Odin/wiki/Keywords-and-Operators

Maybe its more appropriate to have them in a different group, idk.